src/tasks/postLeagueRankings.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { client } from '../discord';
+import { TextChannel } from 'discord.js';
+import { Task } from './types';
+import config from '../config';
+
+const LEAGUE_RANKINGS =
+  'https://spiedie.xyz/rs/league/LeagueRankingProgression';
+
+//const CHANNEL_ID = '267765914085228554';
+
+const postLeagueRankings: Task<undefined, number> = {
```

Comment:
I think there is some formatting errors in this file (with tab spaces).
If you're using VSCode you can install the Prettier plugin and it will work with the prettier config defined by this repository. You can also set it to format on file save so it auto-formats when saving.